fs/nxffs/nxffs_initialize.c

```diff
@@ -54,12 +54,12 @@ const struct mountpt_operations nxffs_operations =
   nxffs_write,       /* write */
   NULL,              /* seek -- Use f_pos in struct file */
   nxffs_ioctl,       /* ioctl */
+  NULL,              /* mmap */
 #ifdef __NO_TRUNCATE_SUPPORT__
```

Comment:
Just note here:
I can't find how `__NO_TRUNCATE_SUPPORT__` can be enabled. Should we clean it up?

Reply:
Look like this is partial work @patacongo left:
https://github.com/apache/nuttx/blob/master/fs/nxffs/README.txt#L187-L191